tests/update/patch_sle.pm

```diff
@@ -235,7 +235,14 @@ sub sle_register {
             # that do not show license agreement during installation but do when registering
             # after install
             set_var('IN_PATCH_SLE', 1);
-            yast_scc_registration();
+            # To use commands to register the product and addons
+            if (get_var('ADDON_REGBYCOM')) {
```

Comment:
Please add comments to let user know some background info and on which scenarios are not suitable to set this setting, such as not support on SLE11.